dev/docs/supported-features.md

```diff
@@ -130,3 +130,12 @@ Supported features are marked with a grey checkbox
 - [x] TCPRoute - we are limited by Consul's ability to only route to a single TCP-based upstream, so TCP-based Gateway listeners only support a single TCPRoute with a single rule with a single backend; otherwise, either the TCPRoute is considered invalid or the Gateway status is set as having conflicting routes.
 - [ ] TLSRoute - TODO
 - [x] UDPRoute *not supported*
+- [x] ReferencePolicy - Routes currently require ReferencePolicies for all BackendRefs, and do not support partial acceptance if any BackendRefs are unpermitted
+  - [x] From
+    - [x] HTTPRoute
+    - [x] TCPRoute
+    - [ ] TLSRoute
+    - [ ] UDPRoute
```

Comment:
Think you can drop these for now just since we have 0 support for either of them in the codebase at the moment, and no active work towards implementing them.

Reply:
I'd lean towards leaving them as they are explicitly noted as `Core` conformance level in the [ReferencePolicy docs](https://gateway-api.sigs.k8s.io/v1alpha2/references/spec/#gateway.networking.k8s.io/v1alpha2.ReferencePolicyFrom)